kong/clustering/init.lua

```diff
@@ -272,17 +65,23 @@ function _M:serve_version_handshake()
 end
 
 function _M:init_worker()
-  self.plugins_list = assert(kong.db.plugins:get_handlers())
-  sort(self.plugins_list, function(a, b)
+  local plugins_list = assert(kong.db.plugins:get_handlers())
+  sort(plugins_list, function(a, b)
     return a.name:lower() < b.name:lower()
   end)
 
-  self.plugins_list = pl_tablex.map(function(p)
+  plugins_list = pl_tablex.map(function(p)
     return { name = p.name, version = p.handler.VERSION, }
-  end, self.plugins_list)
+  end, plugins_list)
 
   local role = self.conf.role
   if role == "control_plane" then
+    self.json_handler = require("kong.clustering.control_plane").new(self)
```

Comment:
is there any reason to move this from `new()` to `init_worker()`?

Reply:
The reason is only code tidy.